node_modules/oae-preview-processor/tests/test-previews.js

```diff
@@ -1003,7 +1003,7 @@ describe('Preview processor', function() {
                         _verifySignedUriDownload(restCtx, content.previews.mediumUrl, function() {
 
                             // Assert that short youtube links are processed as well
-                            _createContentAndWait('link', 'http://youtu.be/lgTQ5I_H4Xk', null, function(restCtx, content) {
+                            _createContentAndWait('link', 'http://youtu.be/lgTQ5I_H4Xk?t=130', null, function(restCtx, content) {
```

Comment:
I had considered it but didn't feel like it was worth it. Not too passionate either way but I just opted to inflate test time as little as possible.